linker/shared/src/main/scala/org/scalajs/linker/analyzer/Infos.scala

```diff
@@ -456,9 +457,9 @@ object Infos {
     private val builder = new ReachabilityInfoBuilder
 
     def generateMethodInfo(methodDef: MethodDef): MethodInfo = {
-      val (_, params, result) = decodeMethodName(methodDef.encodedName)
-      params.foreach(builder.maybeAddReferencedClass)
-      result.foreach(builder.maybeAddReferencedClass)
+      val encodedName = methodDef.encodedName
```

Comment:
Yes, I think we'll need a pass at all the existing `encodedName`s to give them better names, but I'd keep that for a separate PR. I updated the local `val` here anyway, because at least we shouldn't *introduce* more `encodedName`s in this commit.